examples/cxx/README.md

```diff
@@ -1,8 +1,50 @@
 # Instructions
 
+## Install dependencies
+
+Since iceoryx2 is written in Rust we need to install that first.
+We recommend the [official approach](https://www.rust-lang.org/tools/install).
+
+```sh
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Then C/C++ compiler and library dependencies must be installed.
+
+### ArchLinux
+
+```sh
+pacman -Syu --noconfirm clang cmake gcc git
+```
+
+### Ubuntu
+
+```sh
+sudo apt-get update
+sudo apt-get install -y \
+     binutils-dev \
+     build-essential \
+     clang \
+     cmake \
+     curl \
+     flex \
+     gcc \
+     gcc-multilib \
+     g++ \
+     g++-multilib \
+     git \
+     libacl1-dev \
+     libc6-dev \
+     libc6-dev-i386 \
+     libc6-dev-i386-cross \
+     libstdc++6-i386-cross \
+     libdwarf-dev \
+     libelf-dev
+```
```

Comment:
I think it would be better to use a script for this, else it get outdated when the information is duplicated in many places. The CI script is maybe not ideal since it also sets up the additional users. Maybe it can be split up into a script which just installs the required packages on ubuntu and the current CI script would then call that script.

What do you think?